lua/transfer/transfer.lua

```diff
@@ -65,6 +65,7 @@ local function build_scp_path(deployment, remote_file)
   if deployment.username then
     remote_path = remote_path .. deployment.username .. "@"
   end
+  remote_file = string.gsub(remote_file, "\\", "/")
```

Comment:
I'm not sure whether it is no possible case to break something on Linux or Mac. Sometimes "\" can be used in path before spaces, for example. So I suggest wrapping this change in the IF block, where check OS is exactly Windows.